src/everett/manager.py

```diff
@@ -27,16 +27,13 @@
 )
 
 
-# This is a stack of overrides to be examined in reverse order
-_CONFIG_OVERRIDE = []
-
 # Regex for valid keys in an env file
 ENV_KEY_RE = re.compile(r"^[a-z][a-z0-9_]*$", flags=re.IGNORECASE)
 
 logger = logging.getLogger("everett")
 
 
-def qualname(thing):
+def qualname(thing: Any) -> str:
```

Comment:
"qualname" is probably the wrong name for this thing. It should probably be "prettyname" or something like that especially since it defaults to returning repr(thing). I'll look at renaming it in a future PR.